salt/config/__init__.py

```diff
@@ -919,6 +919,14 @@ def _gather_buffer_space():
 
     # Minion data cache driver (one of satl.cache.* modules)
     'cache': str,
+    # Enables a fast in-memory cache booster and sets the expiration time.
+    'memcache_expire_seconds': int,
```

Comment:
Oh, right, it's about retrieving from the cache database, not the caching itself.

Thanks @DmitryKuzmenko